plumber/api/importer.py

```diff
@@ -387,9 +507,20 @@ def import_mdl(
     path: str,
     from_game: bool = True,
     context=None,
-    # MDL-specific options
-    import_animations: bool = True,
+    # General settings
```

Comment:
Based on what was mentioned in the other comment, these should be named with a prefix related to what import type they belong to in the API surface